pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -336,6 +346,12 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(env model.Env
 	return listeners
 }
 
+type listenerEntry struct {
+	protocol     model.Protocol
+	listener     *xdsapi.Listener
+	creationTime time.Time
```

Comment:
this field is not useful here. see comments below

Reply:
It's only used for the log messages